spec/fixture_app/config/application.rb

```diff
@@ -1,5 +1,6 @@
 module Fixture
   class Application < Rory::Application
     turn_off_request_logging!
+    filter_parameters :orcas, :noodles
```

Comment:
Is this a convention you prefer? I would of passed in an array parameters to filter, and they are going to end up in an array anyways. To me it is more clear that I am passing in a collection of things vs going to look up and see the method definition is doing with the first, second, third, etc... params.